lib/Tooling/Refactor/Extract.cpp

```diff
@@ -171,8 +184,12 @@ Optional<CompoundStatementRange> getExtractedStatements(const CompoundStmt *CS,
   assert(Begin && End);
   CompoundStatementRange Result;
   Result.First = std::find(CS->body_begin(), CS->body_end(), Begin);
```

Comment:
Is the first std::find necessary? Is there some reason to not "look into" PseudoObjectExpr's on the first pass through the compound stmt?